DirectEmployers/MyJobs
# [PD-3462] cleanup tab component

## What's this PR do?
PR cleans up the tab component code

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the analytics application or view the code

## How should this be manually tested?
Pull down the repository and go into any analytics application. Make sure the functionality of the tabs in the same along with the app in general or view the code

## Any background context you want to provide?
